content/_partners/our-services._en.md

```diff
@@ -3,6 +3,7 @@ layout: partners/our-services
 permalink: /partners/our-services/
 redirect_from:
  - /docs/login-gov-roadmap-may-2023.pdf/
+ - /docs/login-gov-roadmap-november-2023.pdf/
```

Comment:
Do these redirects actually work? On my computer, they download a PDF for the HTML page, which is invalid. I wonder if we added these just to avoid the build failures on missing URLs?

Reply:
@aduth That's a good point. They don't work without the trailing `/` -- which doesn't help.

I suppose they could be removed. This topic could be reviewed at a later date, perhaps a 301 redirect could be tried at a lower level.